app/src/main/res/values/styles.xml

```diff
@@ -11,12 +11,36 @@
         <item name="android:textAlignment">center</item>
     </style>
 
-    <style name="ButtonStyleMainActivity">
-        <item name="android:backgroundTint">@color/button_main_activity_cream</item>
+    <style name="InputStyleDatePickerReservationActivity">
+        <item name="android:inputType">date</item>
+        <item name="android:focusable">false</item>
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_entry_date_margin_top</item>
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_entry_date_text_size</item>
+        <item name="android:clickable">true</item>
+        <item name="android:maxLines">1</item>
+    </style>
+
+    <style name="InputStyleReservationActivityParkingNumber">
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_parking_number_margin_top</item>
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_parking_number_text_size</item>
+    </style>
+
+    <style name="InputStyleReservationActivityKeyCode">
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_parking_code_text_size</item>
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_parking_code_margin_top</item>
+    </style>
+
+    <style name="FromDateReservationActivity">
+        <item name="android:textSize">@dimen/text_reservation_activity_from_date_text_size</item>
+        <item name="android:layout_marginTop">@dimen/text_reservation_activity_from_date_margin_top</item>
+    </style>
+
+    <style name="ToDateReservationActivity">
+        <item name="android:textSize">@dimen/text_reservation_activity_from_date_text_size</item>
+        <item name="android:layout_marginTop">@dimen/text_reservation_activity_from_date_margin_top</item>
     </style>
```

Comment:
This styles are the same, you can use only one style to define from/to date reservation

Reply:
done!